File: pm_core_1/src/core/domain/entity/issue_type_entity.go
/*
Author: QuanTuanHuy
Description: Part of Serp Project
*/

package entity

type IssueTypeEntity struct {
	BaseEntity

	TenantID       int64   `json:"tenantId"`
	TypeKey        string  `json:"typeKey"`
	Name           string  `json:"name"`
	Description    *string `json:"description,omitempty"`
	IconUrl        *string `json:"iconUrl,omitempty"`
	HierarchyLevel int     `json:"hierarchyLevel"`
	IsSystem       bool    `json:"isSystem"`
}

File: pm_core/src/main/java/serp/project/pmcore/core/domain/entity/IssueTypeEntity.java
/**
 * Author: QuanTuanHuy
 * Description: Part of Serp Project
 */

package serp.project.pmcore.core.domain.entity;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import lombok.experimental.SuperBuilder;

@Getter
@Setter
@SuperBuilder
@NoArgsConstructor
@AllArgsConstructor
public class IssueTypeEntity extends BaseEntity {
    private Long tenantId;
    private String typeKey;
    private String name;
    private String description;
    private String iconUrl;
    private Integer hierarchyLevel;
    private boolean isSystem;
}
